protobuf/python-types/event_msg_pb2.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# source: wrapper.proto
+# source: event_msg.proto
```

Comment:
not a big problem, but I think this should be in the second commit (no need to change it).